### DISCUSSION TITLE
native aot support

### DISCUSSION THREAD

**Discussion by ivanjx** [aaubry/YamlDotNet#739] [Q&A]
does this library supports net 7's native aot with `PublishAot`? thanks

**Comment by jas88** [comment]
The StaticGenerator work on this should enable that, though it seems to have broken in this week's update (I'm getting a `System.NotImplementedException: The method or operation is not implemented.`   at `YamlDotNet.Serialization.StaticContext.GetFactory()`, which looks like the static code generator isn't getting invoked properly any more).

**Comment by EdwardCooke** [comment]
Try it now. I just pushed up an updated package.

**Comment by jas88** [comment]
Thanks @EdwardCooke, that's fixed it.

**Comment by erikrose100** [comment]
I'm getting the same error

**Comment by EdwardCooke** [comment]
Give it a try now, I just published a matching version.
